test/e2e/lib/framework/pageSetups.ts

```diff
@@ -11,26 +11,26 @@ export interface SetupOptions {
   body?: string
 }
 
-export type SetupFactory = (options: SetupOptions) => string
+export type PageFactory = (options: PageSetupOptions, intakeUrl: string) => string
 
 const isBrowserStack =
   browser.config.services &&
   browser.config.services.some((service) => (Array.isArray(service) ? service[0] : service) === 'browserstack')
 
-export const DEFAULT_SETUPS = isBrowserStack
-  ? [{ name: 'bundle', factory: bundleSetup }]
+export const DEFAULT_PAGE_SETUPS = isBrowserStack
+  ? [{ name: 'bundle', pageFactory: bundleSetup }]
```

Comment:
we had `setup`, now we have `page`, `setup` and `pageSetup`.
What are the differences?

Reply:
I renamed `setup` to `pageSetup` to match the file name, which I found clearer.
I renamed `factory` by `pageFactory` mainly because at some point in the code, it's passed as parameter and we loose the context. I could also change the code to pass the pageSetup and keep the context.
Wdyt?

Reply:
Topic for another time. I roll back.